WebService.Test/v1/Models/SimulationApiModel/SimulationApiModelTest.cs

```diff
@@ -143,6 +144,20 @@ private void SetupConnectionStringManager()
                 .Returns(Task.CompletedTask);
         }
 
+        private void SetupRateReporter()
+        {
+            this.rateReporter
+                .Setup(x => x.GetCounters())
+                .Returns(new Simulation.SimulationRateLimits
+                {
+                    ConnectionsPerSecond = It.IsAny<int>(),
```

Comment:
I don't understand this setup. You're specifying htat rateReporter will return a SimulationRateLimits object, but its values are defined is Mock properties... shouldn't we be specifying actual values here for the object that we return?